docs/config/lua/config/quote_dropped_files.md

```diff
@@ -3,25 +3,29 @@
 *Since: nightly builds only*
 
 Controls how file names are quoted (or not) when dragging and dropping.
-There are three possible values:
+There are five possible values:
 
 * `"None"` - no quoting is performed, the file name is passed through as-is.
-* `"SpacesOnly"` - backslash-escape only spaces, leaving all other characters as-is.  This is the default.
+* `"SpacesOnly"` - backslash-escape only spaces, leaving all other characters as-is.  This is the default for non-Windows platforms.
 * `"Posix"` - use POSIX style shell word escaping.
+* `"Windows"` - use Windows style shell word escaping: double-quote filename with space characters in it, and leaving others as-is. This is the default on Windows.
+* `"WindowsAlwaysQuoted"` - like `"Windows"`, while always double-quote the filename.
 
 For example:
 
-|`quote_dropped_files` |file name    |quoted result  |
-|----------------------|-------------|---------------|
-|`"None"`              |`hello world`|`hello world`  |
-|`"SpacesOnly"`        |`hello world`|`hello\ world` |
-|`"Posix"`             |`hello world`|`"hello world"`|
+| `quote_dropped_files`   | file name        | quoted result       |
+|-------------------------|------------------|---------------------|
+| `"None"`                | `hello ($world)` | `hello ($world)`    |
+| `"SpacesOnly"`          | `hello ($world)` | `hello\ ($world)`   |
+| `"Posix"`               | `hello ($world)` | `"hello (\$world)"` |
+| `"Windows"`             | `hello ($world)` | `"hello ($world)"`  |
+| `"WindowsAlwaysQuoted"` | `hello($world)`  | `"hello($world)"`   |
```

Comment:
```suggestion
| `"WindowsAlwaysQuoted"` | `hello ($world)`  | `"hello ($world)"`   |
```

Reply:
Though originally I thought this would better show how this works, it's good to keep consistency for this, and the documentation above should be enough, fixed.